apps/cowswap-frontend/src/modules/usdAmount/updaters/UsdPricesUpdater.tsx

```diff
@@ -29,7 +29,7 @@ const swrOptions: SWRConfiguration = {
   revalidateOnFocus: true,
 }
 
-export function UsdPricesUpdater() {
+export function UsdPricesUpdater({ id }: { id?: string }) {
```

Comment:
I imagine the answer is no, but is there any way to fool/mock swr without adding this test-exclusive parameter?
That doesn't feel right.

In the worst case, make it clear this is used only for testing.

Maybe

```suggestion
export function UsdPricesUpdater({ testCacheIdSuffix }: { testCacheIdSuffix = '' }) {
```